src/Parsec/Shaiya/NpcQuest/BaseNpc.cs

```diff
@@ -46,8 +53,12 @@ protected void ReadBaseNpcSecondSegment(SBinaryReader binaryReader)
             MoveDistance = binaryReader.Read<int>();
             MoveSpeed = binaryReader.Read<int>();
             Faction = (FactionInt)binaryReader.Read<int>();
-            Name = binaryReader.ReadString(false);
-            WelcomeMessage = binaryReader.ReadString(false);
+
+            if (_format <= Format.EP5) // In ep 8, messages are moved to separe translation files.
```

Comment:
This check should be `_format <= Format.EP8` because up until EP7 strings are still part of this file.

Reply:
yep